v22.1/add-super-region.md

```diff
@@ -0,0 +1,115 @@
+---
+title: ADD SUPER REGION
+summary: The ADD SUPER REGION statement adds a super region made up of a set of regions such that data from regional tables will be stored in those regions. 
+toc: true
+docs_area: reference.sql
+---
+
+ The `ALTER DATABASE .. ADD SUPER REGION` [statement](sql-statements.html) adds a [super region](multiregion-overview.html#super-regions) to a [multi-region database](multiregion-overview.html).
+
+{% include enterprise-feature.md %}
+
+{{site.data.alerts.callout_info}}
+`ADD SUPER REGION` is a subcommand of [`ALTER DATABASE`](alter-database.html).
+{{site.data.alerts.end}}
+
+{% include common/experimental-warning.md %}
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-22.1/grammar_svg/alter_database_add_super_region.html %}
+</div>
+
+## Parameters
+
+| Parameter       | Description                                                                                                       |
+|-----------------+-------------------------------------------------------------------------------------------------------------------|
+| `database_name` | The database to which you are adding a [super region](multiregion-overview.html#super-regions).                   |
+| `name`          | The name of the [super region](multiregion-overview.html#super-regions) being added to this database.             |
+| `name_list`     | The names of the [database regions](multiregion-overview.html#database-regions) being added to this super region. |
```

Comment:
s/being added to this super region 

I'd say something like the super region consists of this set of regions

Reply:
Makes sense - updated to "The super region consists of this set of [database regions](multiregion-overview.html#database-regions)"